src/lib.rs

```diff
@@ -5,6 +5,7 @@
 #![allow(clippy::inline_always)]
 #![allow(clippy::module_name_repetitions)]
 #![allow(clippy::similar_names)]
+#![allow(clippy::doc_markdown)]
```

Comment:
Was there some sort of issue that required this to be `allow`ed?

Reply:
This seems like a pretty unreliable lint overall. A lot (>50%) of cases that should have been marked weren't, and it also seems to just mark anything in camelCase or PascalCase as needing backticks. YCbCr was one such victim.